roles/openshift_logging/defaults/main.yml

```diff
@@ -0,0 +1,64 @@
+---
+image_prefix: docker.io/openshift/origin-
+image_version: latest
+logging_use_ops: False
+master_url: https://kubernetes.default.svc.cluster.local
+public_master_url: https://localhost:8443
+logging_namespace: logging
+etcd_generated_certs_dir: /etc/etcd/generated_certs
+install_logging: True
+
+curator_default_days: 30
+curator_run_hour: 0
+curator_run_minute: 0
+curator_run_timezone: UTC
+curator_script_log_level: INFO
+curator_log_level: ERROR
+
+kibana_hostname: kibana.example.com
```

Comment:
Why is this a valid default? Shouldn't it be `kibana.{{ the_given_subdomain }}` ?